workers/unity/Packages/com.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -42,19 +49,72 @@ public static void BuildConfig()
             }
         }
 
-        public static void Launch()
+        public static void LaunchClient()
         {
-            BuildConfig();
+            var command = GetCommand();
+            var commandArgs = GenerateCommandArgs("local worker launch UnityClient default");
 
-            var command = Common.SpatialBinary;
-            var commandArgs = "local launch";
-            
-            if (Application.platform == RuntimePlatform.OSXEditor)
+            var processInfo = new ProcessStartInfo(command, commandArgs)
             {
-                command = "osascript";
-                commandArgs = $"-e 'tell application \"Terminal\"\nactivate\ndo script \"cd {SpatialProjectRootDir} && {Common.SpatialBinary} local launch\"\nend tell'";
+                CreateNoWindow = false,
+                UseShellExecute = true,
+                WorkingDirectory = SpatialProjectRootDir
+            };
+
+            var process = Process.Start(processInfo);
+
+            if (process == null)
+            {
+                Debug.LogError("Failed to start an additional client locally.");
+                return;
             }
 
+            process.EnableRaisingEvents = true;
+            process.Exited += (sender, args) =>
+            {
+                // N.B. This callback is run on a different thread.
+                if (process.ExitCode == 0)
+                {
+                    return;
+                }
+
+                var logPath = Path.Combine(SpatialProjectRootDir, "logs");
+                var latestLogFile = Directory.GetFiles(logPath, "*unityclient.log")
```

Comment:
As far as I'm aware the log will always be called `external-default-unityclient.log`

Reply:
For a standalone client

Reply:
if you specify it to be called that way in the json file

Reply:
Yes but a user could change it from `*unityclient.log` as well. This may want to read the worker json files